packages/apollo-language-server/src/errors/validation.ts

```diff
@@ -20,17 +20,13 @@ import { ToolError, logError } from "./logger";
 export function getValidationErrors(
   schema: GraphQLSchema,
   document: DocumentNode,
-  fragments?: { [fragmentName: string]: FragmentDefinitionNode }
+  fragments?: { [fragmentName: string]: FragmentDefinitionNode },
+  excludeRules: string[] = ["NoUnusedFragments", "KnownDirectives"]
 ) {
-  const specifiedRulesToBeRemoved = [
-    NoUnusedFragmentsRule,
```

Comment:
@martijnwalraven Do you know offhand why this one was needed? 1c22f0680c8f8be9c89782c402fd7413fe8d0e91